src/helpers/checkArgs.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable node/prefer-global/process */
+import { BG_ERR, BG_RESET } from '../rules/asceeCodes'
+
+const checkArgs = (argv: string[], config: { apply?: any, ignore?: any }) => {
```

Comment:
I think you can use this `Config` type from the `/types/Config.ts` for the config parameter
```ts
export interface Config {
  path: string
  apply: string
  ignore: string
  exclude: string
  group: string
  level: string
  sort: string
  output: string
  override: OverrideConfig
  isDefault?: boolean
}
```